pages/03.Setup/01.how-to-install-mautic/03.Installing with Composer/docs.en.md

```diff
@@ -72,7 +72,6 @@ Follow the steps below to update your core files.
 `bin/console cache:clear`
 `bin/console mautic:update:apply --finish`
 `bin/console doctrine:migration:migrate --no-interaction`
-`bin/console doctrine:schema:update --no-interaction --force`
```

Comment:
@nickveenhof @mollux is there any problem with removing this command from the Composer instructions?